### ISSUE TITLE
Purpose of each input field collecting information can be determined

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/structured-metadata-editor-react#72]
### Description
SME audit report line item 1.3.5, referenced in this issue:  #64 

WCAG 2.1 Checkpoint for Identify Input Purpose  - Missing accessible label for input field for seek time

When evaluating the application with WAVE, it gives 1 'Missing form label' error for seek time input field.
When using Chrome VOX screen reader, it could not describe the purpose of each input field, except for identifying them as input fields.

### Done Looks Like
- WAVE extension does not give any errors for 'Missing form labels'.
- Able to identify purpose of each input field using a screen reader

### To Do
- [ ] Implement a solution
- [ ] Re-test

### Notes
This fixes success criterion for,
- WCAG 2.1 Checkpoint 3.3.2 - Labels or Instructions
